recipes-kernel/linux/nilrt-nohz/README.nohz

```diff
@@ -0,0 +1,98 @@
+==============================
+NI Linux Real-Time NOHZ Kernel
```

Comment:
More of a question, but should the kernel version be referenced in the title?

Reply:
I've intentionally left the version out to avoid having to rev this document for every release. A user can easily find out the kernel version using standard Linux tools.